core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -95,11 +96,9 @@
         synchronized void registerPort(int nodeId, int port) {
             controllerPorts.put(nodeId, port);
             if (controllerPorts.size() >= expectedControllers) {
-                future.complete(controllerPorts.entrySet().stream().
-                    collect(Collectors.toMap(
-                        Map.Entry::getKey,
-                        entry -> new RaftConfig.InetAddressSpec(new InetSocketAddress("localhost", entry.getValue()))
-                    )));
+                future.complete(controllerPorts.entrySet().stream().collect(Collectors.toMap(
+                    Map.Entry::getKey,
+                    entry -> new RaftConfig.InetAddressSpec(new InetSocketAddress("localhost", entry.getValue())))));
```

Comment:
I think breaking after `stream()` is on purpose here. At least, I use a similar style when I use Java's Stream API to make the chain of operators more readable. Would be a pity if the formatter would change it.